### ISSUE TITLE
[BUG] Intro is too loud, cannot set loudness or skip it.

### ISSUE THREAD

**Issue by apex-legends-bugs** [apex-legends-bugs/bugs#4]
**Describe the bug**
The intro is unskippable on console and it is too loud.

While the intro is supposed to follow the overall volume setting for the game, there is no way to turn down *just* the intro. You'd have to turn down the game fully, which is particularly stupid given that you are forced to play the game at full volume just to hear enemy foot steps.

The intro is purposefully compressed to make it louder and more annoying. It is specifically made to be much louder than any audio in the game, so lowering the overall volume slider would make in-game audio too quiet. Any audio engineer doing this knows exactly what he or she is doing. There is no excuse for blasting people's ear drums with some lazy jingle that's been put through Waves L3 fifty times. This is bad audio engineering.

This is particularly bad for people prone to migraines. This makes it an accessibility issue.

**To Reproduce**
1. Start the game on Xbox

**Expected behavior**
An intro volume slider must be added to Apex Legends. Additionally, an option to skip the intro video may be added.

**Why could this be happening?**
This is happening because the PM for Apex Legends wanted a flashy intro to grab people's attention. Instead it just annoys them.

**System (please complete the following information):**
 - Platform: Console
 - When exactly the issue happened (exact day and date): All the time

**Additional context**

**Workarounds**
Pull out your headset from the controller while the game is starting
